locha-p2pd/src/main.rs

```diff
@@ -109,6 +110,7 @@ pub fn serialize_message(contents: String) -> Vec<u8> {
     let sum_id: String = datetime.as_secs().to_string() + &text_message;
     sha256.input_str(&sum_id);
 
+    message.from_uid = "16Uiu2HAmKXHPfFVEHVWq6PJXGpyQw83D1cmjrBSQGTnsxG2j5zur".to_string();
```

Comment:
This should be the originator nodes peer ID, you can convert it from the identity that is loaded when the locha-p2pd node starts up.